File: ModuleNotFound/ModuleMapNotFound/ViewController.m
//
//  ViewController.m
//  ModuleMapNotFound
//
//  Created by Vadim Belotitskiy on 10/25/20.
//

#import "ApplicationProtocol.h"
#import "ModuleMapNotFound-Swift.h"
#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    ApplicationModel *model = [[ApplicationModel alloc] init];

    UILabel *label = [[UILabel alloc] init];
    label.translatesAutoresizingMaskIntoConstraints = NO;
    label.text = model.nameObject.name;

    [self.view addSubview:label];

    [NSLayoutConstraint activateConstraints:@[
        [label.centerXAnchor constraintEqualToAnchor:self.view.centerXAnchor],
        [label.centerYAnchor constraintEqualToAnchor:self.view.centerYAnchor],
    ]];
}


@end

File: DependenciesFromSPM/DependenciesFromSPM/ViewController.swift
//
//  ViewController.swift
//  DependenciesFromSPM
//
//  Created by Vadim Belotitskiy on 10/27/20.
//

import UIKit
import Alamofire

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        AF.request("https://httpbin.org/get").response { response in
            if let data = response.data,
               let str = String(data: data, encoding: .utf8)
            {
                print(">>> \(str)")
            }
        }
    }


}

